src/zarr/abc/store.py

```diff
@@ -476,6 +474,9 @@ async def getsize_prefix(self, prefix: str) -> int:
         # on to getting sizes. Ideally we would overlap those two, which should
         # improve tail latency and might reduce memory pressure (since not all keys
         # would be in memory at once).
+        from zarr.core.common import concurrent_map
+        from zarr.core.config import config
```

Comment:
were these linter fixes? I'm curious why the imports were moved as part of this PR?

Reply:
I opted for a separate PR: https://github.com/zarr-developers/zarr-python/pull/3372/files